lib/comfortable_mexican_sofa/extensions/is_taggable.rb

```diff
@@ -8,7 +8,7 @@ module ClassMethods
         # Comfy already define #tags and #tags= methods in some models, so let's call :keywords to the Tag associated
         # instances of a model to avoid naming collisions!!!
         # Creates the appropriate AR associations, scopes and instance methods.
-        def is_taggable
+        def is_taggable # rubocop:disable Style/PredicateName
```

Comment:
I don' t know if I agree with this line. I think this two legacy code above should raises on rubocop, in order to us to fix one day (for example right now, I'm refactoring entirely the hippo_import). But even with that thanks for the contribution :beer: